index.bs

```diff
@@ -1817,7 +1914,7 @@ Build a composed graph up to a given output operand into a computational graph a
         1. Set |graph|.{{MLGraph/[[outputDescriptors]]}}[|name|] to |operand|.{{MLOperand/[[descriptor]]}}.
     1. Set [=this=].{{MLGraphBuilder/[[hasBuilt]]}} to true.
     1. Let |promise| be [=a new promise=] in |realm|.
-    1. Run the following steps [=in parallel=]:
+    1. Enqueue the following steps to |graph|.{{MLGraph/[[context]]}}.{{MLContext/[[timeline]]}}:
```

Comment:
Because now a constant operand value may be a constant tensor, the `build()` steps should check whether a constant tensor is destroyed or not before running the following steps in the context timeline.

Reply:
Good idea. I assume we'll need a separate slot to track. PTAL again.